geoportal/c2cgeoportal_geoportal/lib/lingua_extractor.py

```diff
@@ -661,7 +706,8 @@ def _build_url(self, url: Url) -> Tuple[Url, Dict[str, str], Dict[str, Any]]:
     def _layer_attributes(self, url: str, layer: str) -> Tuple[List[str], List[str]]:
         errors: Set[str] = set()
 
-        request = _Request()
+        request = pyramid.threadlocal.get_current_request()
+        request = _Request() if request is None else request
         request.registry.settings = self.config
```

Comment:
It seems we break webapp settings here as we set the settings on the real request.registry